keepassxc-browser/options/options.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+const SUPPORTERS_URL = 'https://raw.githubusercontent.com/wiki/keepassxreboot/keepassxc-browser/Supporters.md';
```

Comment:
Should we really request a third-party service without notifying the user? This would need an entry in the privacy policy.

Reply:
Good point. It would be strange to make this as an optional "feature", and at the same time we shouldn't make any connections to outside if the user wishes so.

Reply:
But well. I'm all in if we want to take the familiar route and just show a hard-coded list we update in every new release.

Reply:
Just add it to the repository.